public/dorequest.php

```diff
@@ -228,7 +260,18 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
 
     case "gameid":
         $md5 = request()->input('m') ?? '';
-        $response['GameID'] = getGameIDFromMD5($md5);
+        $userAgentService = new UserAgentService();
+        $clientSupportLevel = $userAgentService->getSupportLevel(request()->header('User-Agent'));
+        if ($clientSupportLevel === ClientSupportLevel::Blocked) {
+            $response = [
+                'Status' => 403,
+                'Success' => false,
+                'Error' => "This client is not supported",
+                'GameID' => 0,
```

Comment:
Blocked user agents will receive an error that they should report to the user.
![image](https://github.com/user-attachments/assets/4f2f8e3c-f389-4cfb-9d05-2974f4d54c72)

Reply:
Similar feedback as left elsewhere, I'm wondering if we should use the word "emulator" for player-facing messages.

Reply:
I was going to suggest that not all clients are emulators, but all clients using the Connect API should be emulators. I'm fine with changing the wording.